internal/requester/get.go

```diff
@@ -1,16 +1,21 @@
 package requester
 
 import (
-	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"reflect"
+
+	"code.cloudfoundry.org/jsonry"
 )
 
 func (r Requester) Get(url string, receiver any) error {
-	if reflect.TypeOf(receiver).Kind() != reflect.Ptr {
-		return fmt.Errorf("receiver must be of type Pointer")
+	receiverType := reflect.ValueOf(receiver)
+	switch {
+	case receiverType.Kind() != reflect.Ptr:
```

Comment:
thoughts: I am not sure this check adds much value since the `json.Unmarshal` operation without a pointer will fail with an error with a format similar to the following:

```
error json: Unmarshal(non-pointer file.variable)
```